saga-core/src/main/java/org/apache/servicecomb/saga/core/SQLOperation.java

```diff
@@ -23,18 +23,18 @@
 public class SQLOperation implements Operation {
 
   private final String sql;
-  private final List<String> params;
+  private final List<List<Object>> params;
 
-  public SQLOperation(String sql, List<String> params) {
+  public SQLOperation(String sql, List<List<Object>> params) {
```

Comment:
params are string type. Why do you change it to Object?

Reply:
params may be Integer, Date.  so I think Object may be more suitable than String?